tests/nlu/extractors/test_crf_entity_extractor.py

```diff
@@ -15,7 +15,7 @@ def test_crf_extractor(spacy_nlp, ner_crf_pos_feature_config):
                 "entities": [
                     {"start": 16, "end": 20, "value": "west", "entity": "location"}
                 ],
-                "spacy_doc": spacy_nlp("anywhere in the west"),
+                "text_spacy_doc": spacy_nlp("anywhere in the west"),
```

Comment:
You could simply use the constants here. In that case, we don't have to update the tests again once we change the name again.